### PR TITLE
Connector subproject included only if required *_LOC environment variables are set

### LINKED ISSUE #154: Outline changes to build environment (minor)

### PR DESCRIPTION
**Pull request by fhcarter**
#Fixes #154 

Update readme to include information about changes to `settings.gradle`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,5 +334,24 @@ we have put the relevant configuration in the `exampleConfig` property within th
 Note that the format and contents of this document (at least the type-specific part) are to be understood by the extension source implementation.
 Specifically, this JSON document is the configuration document delivered as part of the [`configureExtension`](#configureExtension_operation) operation.
 
+# Developer Notes
+To develop or build within this environment, we use `gradle` to build.
+
+Some of the connectors require other software to build.
+Generally, these are things that are not available a maven downloads,
```
Comment (namirfa) [inline README.md]: "are not available a maven downloads", should this be "as maven downloads"?

Comment (namirfa) [inline README.md]: "are not available a maven downloads"

Should this say "as maven downloads"?

**Comment by namirfa** [comment]
Sorry, lost internet connection and somehow it duplicated my review. LGTM, just had one typo fix.

File: README.md (hunk 1 above)
Comment (fhcarter) [reply README.md]: Indeed.  Thanks.  Cleaned up a few other wording things as well.

**Pull request merged**